styles.css

```diff
@@ -0,0 +1,190 @@
+body {
+  background-color: #191765;
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  width: 100%;
+}
+
+@media only screen and (min-width: 375px) {
```

Comment:
- [ ] Kindly remove min-width, it's not required for the mobile version as it gives an error for width below 375 px as shown below.
<img width="408" alt="Screenshot 2022-03-27 at 1 11 18 AM" src="https://user-images.githubusercontent.com/70269507/160258658-e736914f-1393-4faf-8a24-c0e61ccac746.png">